src/mapblock.h

```diff
@@ -498,21 +498,14 @@ class MapBlock
 
 	/*
 		Used only internally, because changes can't be tracked
+		Before use check that isValidPosition(x, y. z) returns
+		"true".
 	*/
-
 	inline MapNode &getNodeRef(s16 x, s16 y, s16 z)
 	{
-		if (!isValidPosition(x, y, z))
-			throw InvalidPositionException();
```

Comment:
have you checked whether there are any other callers?

Reply:
Yes. There aren't any more.

Reply:
It is a private method so only mapblock.cpp needed to be checked. But the way I am doing this checking was to rename the method to getNodeRefX and see where else the build fails.